src/components/StudyDesignCriteria.vue

```diff
@@ -263,21 +258,23 @@
           </div>
           <div class="col q-pl-sm">
             <q-input
+              min="0"
               align="center"
               v-model.trim="v.minBMI.$model"
-              hint="Minimum BMI."
-              readonly
-              mask="##.##"
-            />
+              hint="Minimum BMI." mask="##.##"
+              onkeypress="return event.charCode >= 48 && event.charCode <= 57"
+              @blur="v.minBMI.$touch" @input="update()"
+              :error="v.minBMI.$error" error-message="A minimum BMI is required"/>
           </div>
```

Comment:
you can also add the [numeric validator](https://vuelidate.js.org/#sub-builtin-validators) to impose that this is must be a number. Validators are specified in StudyDesign.vue, lines 182 and 183

Reply:
There is already a validator in place, the line 265 `onkeypress="return event.charCode >= 48 && event.charCode <= 57"` makes sure that the user can only input numbers between 0-9.

Reply:
OK